lib/rpc.ts

```diff
@@ -110,11 +111,12 @@ export class Rpc extends EventEmitter implements IForwarderDest {
   constructor(options: {logger?: IRpcLogger, sendMessage?: SendMessageCB,
                         callWrapper?: ICallWrapper} = {}) {
     super();
-    const {logger = console, sendMessage = inactiveSend, callWrapper = plainCall} = options;
+    const {logger = console, sendMessage = null, callWrapper = plainCall} = options;
     this._logger = logger;
-    this._sendMessage = sendMessage;
+    this._sendMessageCB = sendMessage;
     this._callWrapper = callWrapper;
-    this._inactiveQueue = (this._sendMessage === inactiveSend) ? [] : null;
+    this._inactiveQueue = sendMessage ? null : [];
+    this._inactiveSendQueue = [];
```

Comment:
Let's treat this the same as _inactiveRecvQueue, for consistency: either both are null when unused, or both empty array.

Reply:
done